client_connection/manager.go

```diff
@@ -100,6 +102,10 @@ func statusNotConnected() ConnectionStatus {
 	return ConnectionStatus{NotConnected, "", nil}
 }
 
+func statusDisconnecting() ConnectionStatus {
+	return ConnectionStatus{Disconnecting, "", nil}
```

Comment:
Why are we using empty strings and not `nil` in session ids?

Reply:
That's go conventions. But on DTO layer I suggest to have mapping to `null` in JSON value

Reply:
Json encoder can automatically handle empty strings by not rendering fields at all. It's already done by the way.